src/styles/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: 166px;
+  height: 358px;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  color: $main-text-color;
+  padding: 32px 16px 16px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to group CSS properties by type. Here, `width`, `height`, `background-color`, `border-radius`, and `border` are related to the box model and should be grouped together, followed by typography-related properties like `font-family`, `font-size`, `line-height`, and `color`, and finally structural properties like `padding` and `display`. Please regroup them accordingly.